browser/resources/settings/brave_rewards_page/brave_rewards_page.html

```diff
@@ -0,0 +1,134 @@
+  </div>
+  <div class="settings-box continuation">
+    <div class="flex cr-padded-text">
+      <div>$i18n{braveRewardsAutoContributeMonthlyLimitLabel}</div>
+    </div>
+    <settings-dropdown-menu
+      label="$i18n{braveRewardsAutoContributeMonthlyLimitLabel}"
+      pref="{{prefs.brave.rewards.ac.amount}}"
+      menu-options="[[autoContributeMonthlyLimit_]]"
+      disabled="[[!prefs.brave.rewards.ac.enabled.value]]">
+    </settings-dropdown-menu>
+  </div>
+  <div class="settings-box continuation">
+    <div class="flex cr-padded-text">
+      <div>$i18n{braveRewardsAutoContributeMinVisitTimeLabel}</div>
+    </div>
+    <settings-dropdown-menu
+      label="$i18n{braveRewardsAutoContributeMinVisitTimeLabel}"
+      pref="{{prefs.brave.rewards.ac.min_visit_time}}"
+      menu-options="[[autoContributeMinVisitTimeOptions_]]"
+      disabled="[[!prefs.brave.rewards.ac.enabled.value]]">
+    </settings-dropdown-menu>
+  </div>
+  <div class="settings-box continuation">
+    <div class="flex cr-padded-text">
+      <div>$i18n{braveRewardsAutoContributeMinVisitsLabel}</div>
+    </div>
+    <settings-dropdown-menu
+      label="$i18n{braveRewardsAutoContributeMinVisitsLabel}"
+      pref="{{prefs.brave.rewards.ac.min_visits}}"
+      menu-options="[[autoContributeMinVisitsOptions_]]"
+      disabled="[[!prefs.brave.rewards.ac.enabled.value]]">
+    </settings-dropdown-menu>
+  </div>
+  <settings-toggle-button
+    label="$i18n{braveRewardsAutoContributeShowNonVerifiedSitesLabel}"
+    pref="{{prefs.brave.rewards.ac.allow_non_verified}}"
+    disabled="[[!prefs.brave.rewards.ac.enabled.value]]">
+  </settings-toggle-button>
+  <settings-toggle-button
+    label="$i18n{braveRewardsAutoContributeAllowVideoContributionsLabel}"
+    pref="{{prefs.brave.rewards.ac.allow_video_contributions}}"
+    disabled="[[!prefs.brave.rewards.ac.enabled.value]]">
+  </settings-toggle-button>
+  <div class="settings-box continuation">
+    <p>$i18n{braveRewardsAutoContributeDescLabel} <a href="https://brave.com/terms-of-use/" target="_blank" rel="noopener">Terms of Service</a> and <a href="https://brave.com/privacy/browser/" target="_blank" rel="noopener">Privacy Policy</a>.</p></p>
+  </div>
+  <div class="settings-box">
+    <div class="label rewards-primary-title">$i18n{braveRewardsTipButtonsTitle}</div>
+  </div>
+  <settings-toggle-button
+    label="$i18n{braveRewardsInlineTipRedditLabel}"
+    pref="{{prefs.brave.rewards.inline_tip.reddit}}">
+  </settings-toggle-button>
+  <settings-toggle-button
+    label="$i18n{braveRewardsInlineTipTwitterLabel}"
+    pref="{{prefs.brave.rewards.inline_tip.twitter}}">
+  </settings-toggle-button>
+  <settings-toggle-button
+    label="$i18n{braveRewardsInlineTipGithubLabel}"
+    pref="{{prefs.brave.rewards.inline_tip.github}}">
+  </settings-toggle-button>
+</template>
+<template is="dom-if" if="[[!isRewardsEnabled_(prefs.brave.rewards.enabled.value, prefs.brave.brave_ads.enabled.value)]]" restamp>
```

Comment:
Is it possible to just use `prefs.brave.rewards.enabled`?

Reply:
Yes, meant to fix this - thank you!